dashboard/app/models/script.rb

```diff
@@ -35,6 +35,7 @@ class Script < ActiveRecord::Base
 
   include Seeded
   has_many :levels, through: :script_levels
+  has_many :lesson_groups, foreign_key: 'unit_id', dependent: :destroy
```

Comment:
I think we said that we are probably not going to be able to rename script. based on our discussion this morning, I think this means we want to keep using script_id, here and in the db column name. if that doesn't feel right, let's discuss.